### PR TITLE
Update documentation for building docs and running tests

### PR DESCRIPTION
**Pull request by ejeschke**
- updated README.md and install.rst
- also fixed an issue for running tests without `pytest-astropy-header` installed

### CODE REVIEW

**Comment by ejeschke** [comment]
Hi @pllim.  Just updating the docs to remind myself and others how to build the documentation and run the tests. Pls review when you have a minute.

File: doc/install.rst [hunk 1]
```diff
@@ -210,3 +211,29 @@ Ginga.
...
 After installing Anaconda, open the Anaconda Prompt and follow instructions
 under :ref:`install_generic` via ``conda``.
+
+=============
+Running tests
+=============
+
+#. Install the following packages::
+
+    $ pip install pytest pytest-astropy-header
```
Comment (pllim) [inline doc/install.rst]: What about this?
```suggestion
    $ pip install -e .[test]
```

File: doc/install.rst [hunk 2]
```diff
@@ -210,3 +211,29 @@ Ginga.
...
+
+    $ pytest
+
+======================
+Building documentation
+======================
+
+#. Install the following packages::
+
+    $ pip install sphinx-astropy sphinx_rtd_theme
```
Comment (pllim) [inline doc/install.rst]: Maybe like this?
```suggestion
    $ pip install -e .[docs]
```

File: ginga/conftest.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 try:
     from pytest_astropy_header.display import PYTEST_HEADER_MODULES, TESTED_VERSIONS
 except ImportError:
+    TESTED_VERSIONS = {}
```
Comment (pllim) [inline ginga/conftest.py]: Good catch!

File: doc/install.rst (hunk 1 above)
Comment (ejeschke) [reply doc/install.rst]: Interesting. Didn't know about that one. You tested that?

File: doc/install.rst (hunk 2 above)
Comment (ejeschke) [reply doc/install.rst]: Nice! Tested?

File: doc/install.rst (hunk 1 above)
Comment (pllim) [reply doc/install.rst]: https://github.com/ejeschke/ginga/blob/c399f2e2b3bd38857b7a15804a2430e3d0cd7180/.travis.yml#L23

File: doc/install.rst (hunk 2 above)
Comment (pllim) [reply doc/install.rst]: Looks like we also use `recommended` packages for RTD build

https://github.com/ejeschke/ginga/blob/c399f2e2b3bd38857b7a15804a2430e3d0cd7180/.readthedocs.yml#L14-L19

**Comment by ejeschke** [comment]
@pllim, LGTM?

**Pull request merged**